<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Fractals ;)</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="risorse.qrc" >:/new/prefix1/icona-0.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="frame" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Image</string>
      </property>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QToolBox" name="toolBox" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="currentIndex" >
         <number>0</number>
        </property>
        <widget class="QWidget" name="toolFrattali" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>200</width>
           <height>306</height>
          </rect>
         </property>
         <attribute name="label" >
          <string>Fractal</string>
         </attribute>
         <widget class="QDoubleSpinBox" name="frattaliYMin" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>160</y>
            <width>81</width>
            <height>27</height>
           </rect>
          </property>
          <property name="decimals" >
           <number>4</number>
          </property>
          <property name="maximum" >
           <double>20.000000000000000</double>
          </property>
          <property name="minimum" >
           <double>-20.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
          <property name="value" >
           <double>-1.000000000000000</double>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="frattaliYMax" >
          <property name="geometry" >
           <rect>
            <x>100</x>
            <y>160</y>
            <width>81</width>
            <height>27</height>
           </rect>
          </property>
          <property name="decimals" >
           <number>4</number>
          </property>
          <property name="maximum" >
           <double>20.000000000000000</double>
          </property>
          <property name="minimum" >
           <double>-20.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
         <widget class="QLabel" name="label_2" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>140</y>
            <width>186</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text" >
           <string>Ordinate: min and max</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="frattaliXMax" >
          <property name="geometry" >
           <rect>
            <x>100</x>
            <y>110</y>
            <width>81</width>
            <height>27</height>
           </rect>
          </property>
          <property name="readOnly" >
           <bool>false</bool>
          </property>
          <property name="decimals" >
           <number>4</number>
          </property>
          <property name="maximum" >
           <double>20.000000000000000</double>
          </property>
          <property name="minimum" >
           <double>-20.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
         <widget class="QComboBox" name="frattaliDefault" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>60</y>
            <width>171</width>
            <height>23</height>
           </rect>
          </property>
          <item>
           <property name="text" >
            <string>&lt;free></string>
           </property>
          </item>
         </widget>
         <widget class="QDoubleSpinBox" name="frattaliXMin" >
          <property name="geometry" >
           <rect>
            <x>20</x>
            <y>110</y>
            <width>81</width>
            <height>27</height>
           </rect>
          </property>
          <property name="decimals" >
           <number>4</number>
          </property>
          <property name="maximum" >
           <double>20.000000000000000</double>
          </property>
          <property name="minimum" >
           <double>-20.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>0.100000000000000</double>
          </property>
          <property name="value" >
           <double>-1.000000000000000</double>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>0</y>
            <width>181</width>
            <height>51</height>
           </rect>
          </property>
          <property name="title" >
           <string>Image size</string>
          </property>
          <widget class="QSpinBox" name="imgWidth" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>20</y>
             <width>81</width>
             <height>27</height>
            </rect>
           </property>
           <property name="correctionMode" >
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum" >
            <number>1024</number>
           </property>
           <property name="minimum" >
            <number>400</number>
           </property>
           <property name="singleStep" >
            <number>10</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="imgHeight" >
           <property name="geometry" >
            <rect>
             <x>89</x>
             <y>20</y>
             <width>81</width>
             <height>27</height>
            </rect>
           </property>
           <property name="correctionMode" >
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum" >
            <number>768</number>
           </property>
           <property name="minimum" >
            <number>300</number>
           </property>
           <property name="singleStep" >
            <number>10</number>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>90</y>
            <width>179</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text" >
           <string>Abscissa: min and max</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="frattaliC" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>190</y>
            <width>181</width>
            <height>71</height>
           </rect>
          </property>
          <property name="title" >
           <string>Parameter</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
          <widget class="QDoubleSpinBox" name="frattaliCI" >
           <property name="geometry" >
            <rect>
             <x>90</x>
             <y>40</y>
             <width>81</width>
             <height>27</height>
            </rect>
           </property>
           <property name="decimals" >
            <number>4</number>
           </property>
           <property name="maximum" >
            <double>20.000000000000000</double>
           </property>
           <property name="minimum" >
            <double>-20.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.010000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_3" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>20</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text" >
            <string>real and immaginary</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="frattaliCR" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>40</y>
             <width>81</width>
             <height>27</height>
            </rect>
           </property>
           <property name="decimals" >
            <number>4</number>
           </property>
           <property name="maximum" >
            <double>20.000000000000000</double>
           </property>
           <property name="minimum" >
            <double>-20.000000000000000</double>
           </property>
           <property name="singleStep" >
            <double>0.010000000000000</double>
           </property>
           <property name="value" >
            <double>-1.370000000000000</double>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="frattaliOk" >
          <property name="geometry" >
           <rect>
            <x>100</x>
            <y>270</y>
            <width>84</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text" >
           <string>Generate!</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4" >
         <property name="geometry" >
          <rect>
           <x>0</x>
           <y>0</y>
           <width>96</width>
           <height>26</height>
          </rect>
         </property>
         <attribute name="label" >
          <string>Improvments</string>
         </attribute>
         <widget class="QGroupBox" name="aggColore" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>60</y>
            <width>181</width>
            <height>61</height>
           </rect>
          </property>
          <property name="title" >
           <string>Color</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
          <widget class="QComboBox" name="aggColoreSel" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>20</y>
             <width>161</width>
             <height>23</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="aggCorrezioneGrigi" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>0</y>
            <width>181</width>
            <height>51</height>
           </rect>
          </property>
          <property name="title" >
           <string>Gray corrections</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
          <widget class="QLCDNumber" name="aggCorrezioneGrigiDisplay" >
           <property name="geometry" >
            <rect>
             <x>140</x>
             <y>20</y>
             <width>31</width>
             <height>23</height>
            </rect>
           </property>
           <property name="numDigits" >
            <number>3</number>
           </property>
           <property name="segmentStyle" >
            <enum>QLCDNumber::Outline</enum>
           </property>
           <property name="value" stdset="0" >
            <double>1.000000000000000</double>
           </property>
           <property name="intValue" stdset="0" >
            <number>1</number>
           </property>
          </widget>
          <widget class="QSlider" name="aggCorrezioneGrigiVal" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>20</y>
             <width>121</width>
             <height>25</height>
            </rect>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="maximum" >
            <number>100</number>
           </property>
           <property name="singleStep" >
            <number>1</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </widget>
         <widget class="QGroupBox" name="treD" >
          <property name="geometry" >
           <rect>
            <x>10</x>
            <y>130</y>
            <width>181</width>
            <height>131</height>
           </rect>
          </property>
          <property name="title" >
           <string>Range images</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>false</bool>
          </property>
          <widget class="QSpinBox" name="treDScalini" >
           <property name="geometry" >
            <rect>
             <x>40</x>
             <y>40</y>
             <width>61</width>
             <height>27</height>
            </rect>
           </property>
           <property name="buttonSymbols" >
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="maximum" >
            <number>255</number>
           </property>
           <property name="value" >
            <number>5</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="treDAltezza" >
           <property name="geometry" >
            <rect>
             <x>110</x>
             <y>40</y>
             <width>61</width>
             <height>27</height>
            </rect>
           </property>
           <property name="buttonSymbols" >
            <enum>QAbstractSpinBox::PlusMinus</enum>
           </property>
           <property name="maximum" >
            <number>30</number>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
           <property name="value" >
            <number>10</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="treDR" >
           <property name="geometry" >
            <rect>
             <x>20</x>
             <y>90</y>
             <width>50</width>
             <height>27</height>
            </rect>
           </property>
           <property name="correctionMode" >
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum" >
            <number>255</number>
           </property>
           <property name="singleStep" >
            <number>1</number>
           </property>
           <property name="value" >
            <number>120</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="treDG" >
           <property name="geometry" >
            <rect>
             <x>70</x>
             <y>90</y>
             <width>50</width>
             <height>27</height>
            </rect>
           </property>
           <property name="correctionMode" >
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum" >
            <number>255</number>
           </property>
           <property name="value" >
            <number>120</number>
           </property>
          </widget>
          <widget class="QSpinBox" name="treDB" >
           <property name="geometry" >
            <rect>
             <x>120</x>
             <y>90</y>
             <width>50</width>
             <height>27</height>
            </rect>
           </property>
           <property name="correctionMode" >
            <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
           </property>
           <property name="maximum" >
            <number>255</number>
           </property>
           <property name="value" >
            <number>120</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_4" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>20</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text" >
            <string>Step and height</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5" >
           <property name="geometry" >
            <rect>
             <x>10</x>
             <y>70</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text" >
            <string>Wall color (RGB)</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="aggApplica" >
          <property name="geometry" >
           <rect>
            <x>100</x>
            <y>270</y>
            <width>84</width>
            <height>29</height>
           </rect>
          </property>
          <property name="text" >
           <string>Apply!</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Salva_immagine" />
    <addaction name="separator" />
    <addaction name="action_Esci" />
   </widget>
   <widget class="QMenu" name="menu" >
    <property name="title" >
     <string>&amp;?</string>
    </property>
    <addaction name="action_About" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menu" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="action_Salva_immagine" >
   <property name="text" >
    <string>&amp;Save image...</string>
   </property>
  </action>
  <action name="action_Esci" >
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_About" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Help" >
   <property name="text" >
    <string>&amp;Help</string>
   </property>
   <property name="menuRole" >
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="text" >
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>imgWidth</tabstop>
  <tabstop>imgHeight</tabstop>
  <tabstop>frattaliDefault</tabstop>
  <tabstop>frattaliXMin</tabstop>
  <tabstop>frattaliXMax</tabstop>
  <tabstop>frattaliYMin</tabstop>
  <tabstop>frattaliYMax</tabstop>
  <tabstop>frattaliCR</tabstop>
  <tabstop>aggCorrezioneGrigiVal</tabstop>
  <tabstop>aggColoreSel</tabstop>
  <tabstop>treDScalini</tabstop>
  <tabstop>treDAltezza</tabstop>
  <tabstop>treDR</tabstop>
  <tabstop>treDG</tabstop>
  <tabstop>treDB</tabstop>
  <tabstop>aggApplica</tabstop>
  <tabstop>frattaliOk</tabstop>
  <tabstop>frattaliCI</tabstop>
 </tabstops>
 <resources>
  <include location="risorse.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>aggCorrezioneGrigiVal</sender>
   <signal>valueChanged(int)</signal>
   <receiver>aggCorrezioneGrigiDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>570</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel" >
     <x>610</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
